### ISSUE TITLE
GVR uses deprecated UIWebView

### ISSUE THREAD

**Issue by radvani** [googlevr/gvr-ios-sdk#340]
GVR appears to reference UIWebView, which is problematic because iOS is removing UIWebView. Currently submitted applications receive a warning, but eventually will be auto-rejected.

**Comment by toledoal** [comment]
Yes, this is happening to me too. I'm looking to the references of UIWebView but I haven't been able to find those yet. :(

**Comment by cdytoby** [comment]
Happens to me too, any answers?

We're using Unity SDK but the Google VR SDK will be used as a Pod in XCode Project.
